### PR TITLE
Resolve comment for Vert.x Prommetheus procedure

### PR DESCRIPTION
**Pull request by huntley**
Resolves https://issues.jboss.org/browse/RHOARDOC-1492

Late breaking comment from @tsegimont.

### CODE REVIEW

File: docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc [hunk 1]
```diff
@@ -159,9 +159,9 @@ public class CustomVertxApp extends AbstractVerticle {
 
     vertx.setPeriodic(1000, l -> {
       timer.record(() -> {
-
```
Comment (spisiakm) [inline docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: Is this line just indentation fix? :)

Comment (tradej) [reply docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: It seems to me like dangling white space.

Comment (huntley) [reply docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: Hmm. Ok. Excuse me for being dense but is the issue on line 162?

Comment (tradej) [reply docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: @huntley Basically just why you did it.

As a side note, it is considered a semi-bad practice to include unrelated changes like removing dangling whitespace in commits. Is it what happened? (You don't have to edit this PR it if so, just make sure you do white space removal in a separate PR, ideally for the entire doc set at the same time.)

Comment (huntley) [reply docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: Ahh. The spacing change was inadvertent. What is the easiest way to revert that change?

Comment (tradej) [reply docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc]: @huntley Don't worry about it, it's too much work and it's not really "forbidden." Unless you did that deliberately, what I suspect is that your editor's set up to remove dangling white space automatically. If that's the case, I'd only ask you to find that setting and turn it off.

**Pull request merged**